presto-docs/src/main/sphinx/release/release-0.177.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.177
+=============
+
+General
+-------
+
+* Fix correctness issue when performing range comparisons over columns of type ``CHAR``.
+* Fix correctness issue due to mishandling of nulls and non-deterministic expressions in
+  inequality joins unless ``fast_inequality_join`` is disabled.
+* Fix excessive GC overhead caused by lambda expressions. There are still known GC issues
+  with captured lambda expressions pending fix.
+* Prevent duplicate columns in ``CREATE TABLE``.
+* Add support for null values on the left-hand side of a semijoin (i.e., ``IN`` predicate
+  with subqueries).
+* Add ``SHOW STATS`` to display table and query statistics.
+* Add support for preprocessing commands in the Presto CLI.  When the ``PRESTO_PREPROCESSOR``
```

Comment:
Move this to a new "CLI Changes" section, them remove the now-redundant "in the CLI" part.